### DISCUSSION TITLE
How to set size on QR Code?

### DISCUSSION THREAD

**Discussion by HazzeDotCom** [Megabit/Blazorise#4938] [General]
Hi, Im trying the qr code generator, and wants to set the size, what parameter is used for size? I cant find any info about it?

https://blazorise.com/docs/extensions/qrcode 

Im trying to put custom icon in the qr code, but it get too small?

// Hasse

**Comment by stsrki** [comment]
Try the `PixelsPerModule` parameter. Based on it the QRCode will have different sizes.

**Reply by HazzeDotCom** [reply]
thx, that solved my need! :)
